modules/backend/layouts/auth.htm

```diff
@@ -37,6 +37,24 @@
             <link href="<?= $script . '?v=' . $coreBuild; ?>" rel="preload" as="script">
         <?php endforeach; ?>
 
+        <?php if (!Config::get('cms.enableBackendServiceWorkers', false)) : ?>
```

Comment:
@ayumihamsaki sorry for a delay. Small bug 😉 The same in the second example as well. Config gets value if not uses the default, which is passed as the second parameter!)
```suggestion
        <?php if (Config::get('cms.enableBackendServiceWorkers', false)) : ?>
```